glossary.tex

```diff
@@ -201,5 +201,56 @@
 \newglossaryentry{worker-node}
 {
   name={worker node},
-  description={See \gls{compute-node}}
+  description={See \gls{compute-node}},
+}
+\newglossaryentry{OpenStack}
```

Comment:
i would start with a `glossary-clud.tex` file and gather all cloud-specific terminology in there, and `\include` it here
it's most flexible